spec/controllers/users_controller_spec.rb

```diff
@@ -73,8 +73,8 @@
     end
 
     context 'with conferences' do
-      let!(:attendance) { FactoryGirl.create :attendance }
-      let(:user)        { attendance.user }
+      let!(:attendance) { FactoryGirl.create :attendance, :student_attendance }
+      let(:user)        { attendance.team.members.first }
```

Comment:
Minor thing, but since you're looking at a "student", it would be more explicit to use
```ruby
attendance.team.students.first
```
here.